src/main/resources/migrations.xml

```diff
@@ -180,6 +180,19 @@
 			<column name="createdAt" type="datetime" />
 			<column name="updatedAt" type="datetime" />
 		</createTable>
+
+		<createTable tableName="attachments">
+			<column name="id" type="int" autoIncrement="true" >
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="fileName" type="varchar(500)" />
+			<column name="mimeType" type="varchar(500)" />
+			<column name="content" type="varbinary(5000)" />
+			<column name="description" type="varchar(5000)" />
+			<column name="classificationUuid" type="varchar(500)" />
+			<column name="relatedObjectType" type="varchar(500)" />
+			<column name="relatedObjectUuid" type="varchar(500)" />
```

Comment:
Should be:
```
                        <!-- The next two columns combined act as a generic foreign key reference to a (table name, uuid)-tuple -->
                        <column name="relatedObjectType" type="varchar(255)">
                                <constraints nullable="false" />
                        </column>
                        <column name="relatedObjectUuid" type="${uuid_type}">
                                <constraints nullable="false" />
                        </column>
```
as elsewhere, and actually these should be in a separate m2m table (`attachmentRelatedObjects`) linking attachments to other ANET objects.